src/overlays/actors/ovl_En_Horse_Normal/z_en_horse_normal.h

```diff
@@ -6,9 +6,31 @@
 
 struct EnHorseNormal;
 
+typedef void (*EnHorseNormalActionFunc)(struct EnHorseNormal*, GlobalContext*);
+
 typedef struct EnHorseNormal {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x1DC];
+    /* 0x014C */ s32 type;
+    /* 0x0150 */ s32 animationIdx;
+    /* 0x0154 */ PSkinAwb skin;
+    /* 0x01E4 */ u16 unk_1E4;
+    /* 0x01E6 */ char unk_1E6[0x02];
+    /* 0x01E8 */ Vec3f unk_1E8;
+    /* 0x01F4 */ Vec3f unk_1F4;
+    /* 0x0200 */ s32 unk_200;
+    /* 0x0204 */ Vec3f unk_204;
+    /* 0x0210 */ char unk_210[0x08];
+    /* 0x0218 */ f32 unk_218;
+    /* 0x021C */ s16 unk_21C;
+    /* 0x021E */ s16 unk_21E;
+    /* 0x0220 */ f32 unk_220;
+    /* 0x0224 */ char unk_224[0x04];
+    /* 0x0228 */ ColliderCylinder bodyCollider;
+    /* 0x0274 */ ColliderJntSph headCollider;
+    /* 0x0294 */ ColliderJntSphItem headColliderItems[1];
+    /* 0x02D4 */ ColliderCylinder cloneCollider;
```

Comment:
I kind of see why `cloneCollider` is clone-only related (updated in Draw for type 3 only),
but I don't see why `bodyCollider`/`headCollider` are body/head?

Reply:
The original horses (not cloned) have two hitboxes: a cylinder for the body and a sphere for the head. On the other hand, the clones have one giant cylinder hitbox.

Reply:
But how do you know they are for head and body?

Reply:
A mixture of gz hitbox viewer and testing in debug rom changing the sizes of the colliders.